Sources/SwiftParser/Expressions.swift

```diff
@@ -2361,17 +2361,28 @@ extension Parser {
     do {
       var keepGoing: RawTokenSyntax? = nil
       var loopProgress = LoopProgressCondition()
+      var unexpectedPrePatternCase: RawUnexpectedNodesSyntax? = nil
       repeat {
         let (pattern, whereClause) = self.parseGuardedCasePattern()
         keepGoing = self.consume(if: .comma)
         caseItems.append(
           RawSwitchCaseItemSyntax(
+            unexpectedPrePatternCase,
             pattern: pattern,
             whereClause: whereClause,
             trailingComma: keepGoing,
             arena: self.arena
           )
         )
+
+        if keepGoing != nil, let caseToken = self.consume(if: .keyword(.case)) {
+          unexpectedPrePatternCase = RawUnexpectedNodesSyntax(
+            elements: [RawSyntax(caseToken)],
+            arena: self.arena
+          )
+        } else {
+            unexpectedPrePatternCase = nil
```

Comment:
Can you change indentation to 2 spaces by running swift-format (https://github.com/apple/swift-syntax/blob/main/CONTRIBUTING.md#formatting)?

Reply:
Done! Sorry about the sloppiness.

Reply:
Not a problem at all.